api/Trace/Attributes.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Trace;
+
+interface Attributes extends \IteratorAggregate, \Countable
+{
+    /**
+     * Setting event should not invalidate nor change any existing iterators.
+     * @param string $name
+     * @param $value
```

Comment:
type hinting should be added here, and should be the same with Attribute::getValue()

Reply:
Good point.